projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/ReferralService.kt

```diff
@@ -70,7 +70,15 @@ class ReferralService(
                     ContactDetails(
                         date = details.bookingMadeAt,
                         type = ContactTypeCode.BOOKING_MADE,
-                        notes = "To view details of the Approved Premises booking, click here: ${details.applicationUrl}",
+                        notes = listOfNotNull(
+                            "Current: ${DeliusDateFormatter.format(details.arrivalOn)} to ${
+                                DeliusDateFormatter.format(
+                                    details.departureOn
+                                )
+                            }",
+                            "To view details of the Approved Premises booking, click here: ${details.applicationUrl}"
+                        )
+                            .joinToString(System.lineSeparator() + System.lineSeparator()),
```

Comment:
Neither of these strings can be null, so might be clearer as a multi-line string?
```suggestion
                        notes = """
                            Current: ${DeliusDateFormatter.format(details.arrivalOn)} to ${DeliusDateFormatter.format(details.departureOn)}
                            
                            To view details of the Approved Premises booking, click here: ${details.applicationUrl}
                            """.trimIndent(),
```